Altis_Life.Altis/config/Config_Loadouts.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+    class PLAYERSIDE {
+        loadOut[] = {
+            "", // Uniform - empty ("" or {""} or {}) will load nothing...Otherwise, use "" for a unique uniform or {"",""} for a randomly selected uniform
+            "", // Vest - empty ("") will load nothing
+            "", // Backpack - empty ("") will load nothing
+            "", // Weapon - empty ("") will load nothing
+            {{"",0}}, // Mags, amount - empty ({"",0}) will load nothing
+            {} // Specific items to be loaded on player - empty ({}) will load nothing...Otherwise, {"ItemName", Amount}
+        };
+    };
+*/
+class Loadouts {
+    commonItems[] = {"ItemMap","ItemCompass","ItemWatch"}; // Common link items between sides - empty ({} or {""}) will load nothing
```

Comment:
Having common linked items is fine, but there needs to be linked items for each faction as for many they may differ.